app/services/events/create_service.rb

```diff
@@ -36,5 +37,22 @@ def call
     private
 
     attr_reader :organization, :params, :timestamp, :metadata
+
+    def produce_kafka_event(event)
+      return unless ENV['LAGO_KAFKA_BOOTSTRAP_SERVERS'].present?
```

Comment:
Rails/Blank: Use `if ENV['LAGO_KAFKA_BOOTSTRAP_SERVERS'].blank?` instead of `unless ENV['LAGO_KAFKA_BOOTSTRAP_SERVERS'].present?`.